===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(swPipeUnsock_create:object)) at 6 (line 62, column 9) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/swoole-src/src/pipe/PipeUnsock.c
+++ b/table1/benchmarks/swoole-src/src/pipe/PipeUnsock.c
@@ -61,6 +61,7 @@ int swPipeUnsock_create(swPipe *p, int blocking, int protocol)
     ret = socketpair(AF_UNIX, protocol, 0, object->socks);
     if (ret < 0)
     {
+        free(object);
         swWarn("socketpair() failed. Error: %s [%d]", strerror(errno), errno);
         return SW_ERR;
     }
